LICENSE

```diff
@@ -175,7 +175,7 @@
 
    END OF TERMS AND CONDITIONS
 
-   Copyright 2016-2018 MayaData, Inc.
+   Copyright 2016-2019 MayaData, Inc.
```

Comment:
@pranjalv9 -- thanks for catching this! Can you modify this as:

`Copyright 2016-2019 OpenEBS Authors`

Reply:
@kmova i'll do that but can u help in solving error i get during commits